streams/block/block_factory.h

```diff
@@ -20,6 +20,12 @@
 #include "ciphers/seed/seed_block.h"
 #include "ciphers/idea/idea_block.h"
 #include "ciphers/cast/cast_block.h"
+#include <streams/block/ciphers/kasumi/kasumi_factory.h>
+#include <streams/block/ciphers/kuznyechik/kuznyechik_factory.h>
+#include <streams/block/ciphers/misty1/misty1_factory.h>
+#include <streams/block/ciphers/noekeon/noekeon_factory.h>
+#include <streams/block/ciphers/shacal2/shacal2_factory.h>
+#include <streams/block/ciphers/xtea/xtea_factory.h>
```

Comment:
Different style of includes, but we will refactor it later with clang-format inspection.